引用できない資料に対して `urn` 関数を呼び出すのはエラーです。

```julia
urntype(_, x)

```
